src/components/PackageList/PackageList.tsx

```diff
@@ -13,21 +13,22 @@ interface Props {
 }
 
 export const PackageList: React.FC<Props> = props => {
-  const renderPackages: () => React.ReactElement<HTMLElement>[] = () => {
-    return props.packages.map((pkg, i) => {
+  const { packages } = props;
+  const renderPackages: () => ReactNode[] = () => {
+    return packages.map((pkg, i) => {
```

Comment:
This is only a suggestion, but I would avoid creating another constant by doing this:
```suggestion
    return packages.map(({ name, version, description, time, keywords, dist, homepage, bugs, author }, i) => {
```

Reply:
ohh yes! done